metadeploy/api/jobs.py

```diff
@@ -308,12 +309,14 @@ def create_scratch_org(*, plan_id, org_name, result_id):
         # Start installation job automatically if both:
         # - Plan has no preflight
         # - All plan steps are required
-        Job.objects.create(
-            user=None,
-            plan=plan,
-            organization_url=org_config.instance_url,
-            org_id=scratch_org_config.config["org_id"],
-        )
+        with transaction.atomic():
+            job = Job.objects.create(
+                user=None,
+                plan=plan,
+                organization_url=org_config.instance_url,
+                org_id=scratch_org_config.config["org_id"],
+            )
+            job.steps.set(plan.steps.all())
```

Comment:
@wlonk Why this instead of passing in `steps=plan.steps.all()` or something in the `create()`?